articles/article-025-we-all-hate-bugs.md

```diff
@@ -0,0 +1,22 @@
+<!-- BEGIN TITLE -->
+# We all hate Bugs
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+Bugs creep into your house without notice, there they eat your food and clog your sink. 
+They hide in corners and behind large furniture and are most of the times hard to find. If you find a bug that was bothering you for a long time the feeling of success is rewarding.
+A bug in software is very simmilar. It's usually a behavior of the software which is not intended. It is often very hard to find bugs in software and can sometimes be attributed to one single character that was wrongly placed. The act of finding bugs is called Debugging. Like pest control, the code is searched and cleaned from every bug that influences the behavior negatively.
```

Comment:
It's usually -> it is a  ...
or could bugs be other things than behavior, which is not intended?